kafka/modis_kernels_kf.py

```diff
@@ -229,21 +230,22 @@ def get_mcd43_prior(mcd43_fstring, band):
                 gdal.Open(os.path.join(the_dir, "brdf_2016_b06.vrt")),
                 gdal.Open(os.path.join(the_dir, "brdf_2016_b07.vrt")))
 
-    output_plethora = create_output_file(g, the_dir, "test")
+    output_plethora = create_output_file(g, the_dir, "band{}".format(band))
     
-    kf = MODISKernelLinearKalman(modis_obs, days, output_plethora, [] )
+    kf = MODISKernelLinearKalman(modis_obs, days, output_plethora, [], the_band=band )
```

Comment:
I know in the other code you use 0 and 1 for red/NIR but now we are looking at modis bands 1 to 6